R/chillkernel.R

```diff
@@ -2,10 +2,10 @@
 #'  
 #' The function produces a matrix of the estimated density (z) of yield (y) and chill (x). 
 #' As the density function restricts the shape of the kernel to a bivariate normal kernel, it looks slightly different compared to the scatter plot estimates in the pasitR::chillscatter() function.
-#' Density surface plot of chill portions (x) and yield (y). The legend shows the value for the estimated density (z).
+#' Density surface plot of Chill Portions (x) and yield (y). The legend shows the value for the estimated density (z).
 #' 
-#' @param chill is a list of observed annual chill portions corresponding to another list with annual yields. 
-#' @param yield is a list of observed annual yields corresponding to another list with annual chill portions. 
+#' @param chill is a list of observed seasonal Chill Portions corresponding to another list with annual yields. 
```

Comment:
I think 'seasonal' is a good choice here. It is more clear that this is about a cropping cycle.